portfolio/src/main/java/com/google/sps/servlets/ViewCommentsServlet.java

```diff
@@ -21,39 +7,41 @@
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
 import com.google.gson.Gson;
-import com.google.sps.data.Task;
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.sps.data.Comment;
 
-/** Servlet responsible for listing tasks. */
-@WebServlet("/list-tasks")
-public class ListTasksServlet extends HttpServlet {
+/** Servlet responsible for listing comments. */
+@WebServlet("/view-comments")
+public class ViewCommentsServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Task").addSort("timestamp", SortDirection.DESCENDING);
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
-    List<Task> tasks = new ArrayList<>();
+    List<Comment> comments = new ArrayList<>();
     for (Entity entity : results.asIterable()) {
       long id = entity.getKey().getId();
-      String title = (String) entity.getProperty("title");
+      String userComment = (String) entity.getProperty("user_comment");
       long timestamp = (long) entity.getProperty("timestamp");
-
-      Task task = new Task(id, title, timestamp);
-      tasks.add(task);
+      double sentiment = 0;
```

Comment:
another option would be to preserve the null sentiment and return it to the UI (although you have to work with Gson a bit to get it to accept nulls), and then have the UI render those differently. Up to you if you want to go through that exercise, though.

Reply:
Like if it's null, print something like "unknown sentiment" instead of 0?

Reply:
yes, or "n/a", or whatever